server/src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+spring.application.name=server
```

Comment:
You can remove this file as we will use `.yml` files rather the `.properties` for configuration.